vmck/settings.py

```diff
@@ -15,8 +15,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
Let's keep both backends. If `POSTGRES_DATABASE` is set (and yes, the database name should be configurable, not hardcoded to `vmck`) then use `django.db.backends.postgresql`, otherwise fall back to the old setup with `django.db.backends.sqlite3`.